packages/cozy-konnector-libs/src/libs/saveBills.spec.js

```diff
@@ -76,3 +77,120 @@ describe('saveBills', function() {
     ])
   })
 })
+
+describe('appendContractsDatas', function() {
+  it('Append contracts datas if contractId present in options', async () => {
```

Comment:
Indentation is not the same throughout the file, do you have a linter in your editor ?